sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInfo.scala

```diff
@@ -9,8 +9,8 @@ import data.{Ref, Relation}
 
 final class PackageInfo private[daml] (
     val mainPackageId: Ref.PackageId,
-    val mainPackageMetadata: Ast.PackageMetadata,
-    val pkgSignature: Map[Ref.PackageId, Ast.GenPackage[_]],
+    val mainPackage: Ast.Package,
+    val pkgSignature: Map[Ref.PackageId, Ast.Package],
```

Comment:
@remyhaemmerle-da I needed this change since the previous definition wasn't useful for me 🤦 because `Ast.Package =!= Ast.GenPackage[_]` (the rhs is type-widened). I think it's fine to restrict this definition so as it seems that no usages are complaining

Reply:
Why do you need `Ast.Package` ?